### PR TITLE
Avoid revoking lines corresponding to $0 refunds

### PR DESCRIPTION
**Pull request by rlucioni**
Refunds corresponding to a total credit of $0 require no revocation. This change also prevents deadlocking with the LMS which occurs when Otto attempts to revoke an automatically-approved refund. For more context, please see [XCOM-424](https://openedx.atlassian.net/browse/XCOM-424).

@jimabramson or @clintonb, please review.

### CODE REVIEW

File: ecommerce/extensions/fulfillment/tests/test_api.py [hunk 1]
```diff
@@ -90,6 +90,19 @@ def test_revoke_fulfillment_for_refund(self):
         self.assertEqual(refund.status, REFUND.PAYMENT_REFUNDED)
         self.assertEqual(set([line.status for line in refund.lines.all()]), {REFUND_LINE.COMPLETE})
 
+    def test_suppress_revocation_for_zero_dollar_refund(self):
+        """
+        Verify that the function does not require use of fulfillment modules to mark lines in a refund
+        corresponding to a total credit of $0 as complete.
```
Comment (jimabramson) [inline ecommerce/extensions/fulfillment/tests/test_api.py:97]: to explicitly reinforce what the docstring says, could you override settings in this method with `FULFILLMENT_MODULES=[]` ?

**Comment by jimabramson** [comment]
@rlucioni lgtm, only request is the settings override.

File: ecommerce/extensions/fulfillment/tests/test_api.py (hunk 1 above)
Comment (rlucioni) [inline ecommerce/extensions/fulfillment/tests/test_api.py:97]: Updated.

**Comment by jimabramson** [comment]
:+1:

**Pull request merged**